.gitignore

```diff
@@ -1,4 +1,3 @@
 /target
 /iroh_gateway/test_files/*
 .env
-Cargo.lock
```

Comment:
I'm adding Cargo.lock b/c the WASM CI job depends on this file existing.

Reply:
That seems odd, we can just generate it in CI